sdk/identity/azure-identity/setup.py

```diff
@@ -51,7 +51,7 @@
     author_email="azpysdkhelp@microsoft.com",
     url="https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity",
     classifiers=[
-        "Development Status :: 5 - Production/Stable",
+        'Development Status :: 5 - Production/Stable',
```

Comment:
Unintended side effect of script or regex?

Reply:
Yes and No. Auto-versioning tool updates classifier if it doesn't match and keep everything with single quote. Do you want this to be updated to old one? It's anyway going to be updated when version bump is triggered automatically.

Reply:
We format with black, which will change this back to double quotes and allows no configuration of that behavior. I suggest our auto-versioning tool should use double quotes as well.

Reply:
@chlowell : I have fixed versioning tool as well as setup.py files as per your suggestion.